packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.ts

```diff
@@ -284,8 +283,11 @@ export const usePickerValue = <
               props: { ...props, value: action.value, timezone },
             });
 
+      const shortcut = action.name === 'setValueFromShortcut' ? action.shortcut ?? null : null;
```

Comment:
I found this writing easier to read, is it the same, or did I miss a subtility?
```suggestion
      const shortcut = action.name === 'setValueFromShortcut' && action.shortcut ?? null;
```

Reply:
I used the following:

```suggestion
      const shortcut = (action.name === 'setValueFromShortcut' && action.shortcut) || null;
```

With `??` we can end up with a `false` valeu